elasticsearch/README.md

```diff
@@ -108,7 +108,8 @@ helm install --name elasticsearch elastic/elasticsearch --version 7.2.0 --set im
 | `ingress`                  | Configurable [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) to expose the Elasticsearch service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                            | `enabled: false`                                                                                                          |
 | `schedulerName`            | Name of the [alternate scheduler](https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/#specify-schedulers-for-pods)                                                                                                                                                                          | `nil`                                                                                                                     |
 | `masterTerminationFix`     | A workaround needed for Elasticsearch < 7.2.0 to prevent master status being lost during restarts [#63](https://github.com/elastic/helm-charts/issues/63)                                                                                                                                                                  | `false`                                                                                                                   |
-| `lifecycle`                 | Allows you to add lifecycle configuration. See [values.yaml](./values.yaml) for an example of the formatting.                                                                                                                           | `{}`                                                                                                                      |
+| `lifecycle`                 | Allows you to add lifecycle configuration. See [values.yaml](./values.yaml) for an example of the formatting.                                                                                                                                                                                                             | `{}`                                                                                                                      |
+| `sysctlInitContainer.enabled` | Allows you to disable the sysctlInitContainer if you are not allowed to run privileged:true.                                                                                                                                                                                                                            | `false`                                                                                                                   |
```

Comment:
> if you are not allowed to run privileged:true.

This is only one of the reasons why you might want to disable this. You might already have this set on the node level, or you might (when an option is added) be using the default built in sysctl integration that Kubernetes offers. I think this part should just be removed to avoid any confusion.

Reply:
Could you also move it next to the `sysctlVmMaxMapCount` value in this readme and in values.yaml

Reply:
done!